test/integration_test.py

```diff
@@ -558,13 +560,14 @@ def _assert_catalog_complete(self,
     entity_types = ['files', 'projects', 'samples', 'bundles']
 
     def _assert_catalog_empty(self, catalog: CatalogName):
-        with self.subTest('catalog_empty', catalog=catalog):
-            hit_counts = {
-                entity_type: len(self._get_entities(catalog, entity_type))
-                for entity_type in self.entity_types
-            }
-            log.info('Hit counts are %r', hit_counts)
-            self.assertEqual(0, sum(hit_counts.values()))
+        for entity_type in self.entity_types:
+            with self.subTest('catalog_empty',
+                              catalog=catalog,
+                              entity_type=entity_type):
+                hits = self._get_entities(catalog, entity_type)
+                self.assertEqual([],
+                                 [hit['entryId'] for hit in hits],
+                                 f'`{entity_type}` in catalog `{catalog}`')
```

Comment:
https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#string-interpolation

Reply:
But remove since sub test parameters should be logged by test framework.

Reply:
> But remove since sub test parameters should be logged by test framework.

Yea, I made the change since when I ran it through the debug utility in PyCharm it looked different. 
<img width="934" alt="Screen Shot 2020-11-09 at 10 52 30 AM" src="https://user-images.githubusercontent.com/19215510/98583890-f3b45100-2279-11eb-90b9-ec5ea4190189.png">


But the `make integration_test` target looks as you described (output in the [next comment](https://github.com/DataBiosphere/azul/pull/2385#issuecomment-724206524)).

Reply:
Please include a screenshot with the "test" node in the tree expanded and one of the sub tests underneath it selected.

Reply:
Here it is. 
<img width="942" alt="Screen Shot 2020-11-09 at 1 51 02 PM" src="https://user-images.githubusercontent.com/19215510/98600592-aa243000-2292-11eb-986b-2dc96bfc1f7e.png">

Reply:
That screenshot shows that the subtest entry includes catalog and entity type while the console output fragment associated with the entry shows the missing entity IDs.  IOW, there is no need to include this line, neither when running tests on the command line or from Python.